src/modules/copyFiles.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+const fs = require('fs');
+
+const copyFiles = (command, fromFileDirectory, destinationFileDirectory) => {
+  if (command === 'cp' && fromFileDirectory !== destinationFileDirectory) {
+    let data = '';
+
+    try {
+      data = fs.readFileSync(fromFileDirectory, 'utf-8');
+    } catch (error) {
+      global.console.log('Unable to read file, no such file of directory');
+    }
```

Comment:
After catching the error here, the code is continuing running. as a suggestion we can add `return;`

Reply:
agree

Reply:
The linter deleted it as an unnecessary return...